layers/meta-balena-5x-owa/recipes-bsp/u-boot/u-boot-owasys.bbappend

```diff
@@ -6,10 +6,13 @@ inherit resin-u-boot
 FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
 
 # resin-u-boot class patch is rebased
-SRC_URI:remove = " file://resin-specific-env-integration-kconfig.patch "
+SRC_URI:remove = " file://resin-specific-env-integration-kconfig.patch \
+                   git://source.codeaurora.org/external/imx/uboot-imx.git;protocol=https;branch=${SRCBRANCH} \
+"
 
-SRC_URI:append:owa5x = " \
+SRC_URI:append:owa5x = " git://github.com/nxp-imx/uboot-imx.git;protocol=https;branch=${SRCBRANCH} \
```

Comment:
why isn't this done from the BSP layer instead?

Reply:
Indeed our last commit on the BSP meta layer has that change incorporated, but upgrading the meta-owasys will result in the meta-balena-5x-owa patches became useless.

Reply:
I am currently creating an image based on our latest BSP meta-layer, which will save this changes. I'll let you know ASAP